service/src/main/java/tds/config/web/resources/TimeLimitsResource.java

```diff
@@ -0,0 +1,19 @@
+package tds.config.web.resources;
+
+import org.springframework.hateoas.ResourceSupport;
+import tds.config.TimeLimits;
+
+/**
+ * A HATEOAS representation of the {@link tds.config.TimeLimits} for return to the consumer.
+ */
+public class TimeLimitsResource extends ResourceSupport {
```

Comment:
I pulled out the `ResourceSupport`, but I ended up putting it back in.  Since a `TimeLimitConfiguration` is a resource, it needs a link to itself.  I think extending `ResourceSupport` is the easiest way to do so (could be wrong, though).  Alternately, I suppose it would be possible to build a `Resource<TimeLimitConfiguration>` in the controller and add the self link by hand.